File: src/main/java/net/atos/zac/zoeken/model/ZoekObject.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.zoeken.model;

import net.atos.zac.zoeken.model.index.ZoekObjectType;

public interface ZoekObject {

  String IS_TOEGEKEND_FIELD = "isToegekend";

  String getObjectId();

  ZoekObjectType getType();
}

File: src/main/kotlin/net/atos/zac/search/model/zoekobject/ZoekObject.kt
/*
 * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.search.model.zoekobject

interface ZoekObject {
    companion object {
        const val IS_TOEGEKEND_FIELD: String = "isToegekend"
    }

    fun getObjectId(): String

    fun getType(): ZoekObjectType
}
